capture-top-level-project/gradle-enterprise-custom-user-data.groovy

```diff
@@ -0,0 +1,6 @@
+// This script is meant to be used with the common-custom-user-data-maven-extension.
+// It needs to be placed in .mvn/gradle-enterprise-custom-user-data.groovy in the root project directory.
+// (see https://docs.gradle.com/enterprise/maven-extension/#using_the_common_custom_user_data_maven_extension)
+
+buildScan.value('topLevelProject.name', session.topLevelProject.name)
+buildScan.value('topLevelProject.artifactId', session.topLevelProject.artifactId)
```

Comment:
⭕️ Should this also be added to the common custom user data maven extension?

Reply:
I think it should be either here or there. I'm not yet convinced this is useful for all customers so I added it as an "extension".